src/uwtools/cli.py

```diff
@@ -248,6 +251,60 @@ def _dispatch_config_validate(args: Args) -> bool:
     )
 
 
+# Mode execute
+
+
+def _add_subparser_execute(subparsers: Subparsers) -> ModeChecks:
+    """
+    Subparser for mode: execute
+
+    :param subparsers: Parent parser's subparsers, to add this subparser to.
+    """
+    parser = _add_subparser(subparsers, STR.execute, "Execute external driver.")
+    required = parser.add_argument_group(TITLE_REQ_ARG)
+    _add_arg_module(required)
+    _add_arg_classname(required)
+    _add_arg_task(required)
+    _add_arg_schema_file(required)
+    optional = _basic_setup(parser)
+    _add_arg_module_dir(optional)
+    _add_arg_config_file(optional)
+    _add_arg_cycle(optional)
+    _add_arg_leadtime(optional)
+    _add_arg_batch(optional)
+    _add_arg_dry_run(optional)
+    _add_arg_graph_file(optional)
+    _add_arg_key_path(
+        optional,
+        helpmsg="Dot-separated path of keys leading through the config "
+        "to the driver's configuration block",
```

Comment:
I realize this is also similar to the line [here](https://github.com/ufs-community/uwtools/blob/main/src/uwtools/cli.py#L902) in the `Support` methods.  

Should we update so it looks more closely to the `Mode Config` methods [here](https://github.com/ufs-community/uwtools/blob/main/src/uwtools/cli.py#L156)?

`_add_arg_key_path(optional, helpmsg="Dot-separated path of keys to the block to be output")`